tensorflow/tools/ci_build/builds/pip.sh

```diff
@@ -119,7 +127,10 @@ echo "whl file path = ${WHL_PATH}"
 # Install, in user's local home folder
 echo "Installing pip whl file: ${WHL_PATH}"
 
+# Call pip install twice, first time with --upgrade and second time without it
+# This addresses the sporadic test failures related to protobuf version
 ${PYTHON_BIN_PATH} -m pip install -v --user --upgrade ${WHL_PATH} &&
+${PYTHON_BIN_PATH} -m pip install -v --user ${WHL_PATH} &&
```

Comment:
If it means what I think it does, should we just add `set -e` to the top of the file and remove all these?